### ISSUE TITLE
[Crash] Mannequin

### ISSUE THREAD

**Issue by Iomega0318** [ServUO/ServUO#4970]
If you choose "Compare with item in slot" from the context menu and then click the Mannequin the server will crash, don't ask me why I clicked him I just did.

`Exception:
System.InvalidCastException: Unable to cast object of type 'Server.Mobiles.Mannequin' to type 'Server.Item'.
   at Server.Mobiles.Mannequin.CompareWithItemInSlotEntry.InternalTarget.OnTarget(Mobile from, Object targeted) in C:\ServUO\Scripts\Mobiles\NPCs\Mannequin\Mannequin.cs:line 323
   at Server.Targeting.Target.Invoke(Mobile from, Object targeted) in C:\ServUO\Server\Targeting\Target.cs:line 272
   at Server.Network.PacketHandlers.TargetResponse(NetState state, PacketReader pvSrc) in C:\ServUO\Server\Network\PacketHandlers.cs:line 1343
   at Server.Network.MessagePump.HandleReceive(NetState ns) in C:\ServUO\Server\Network\MessagePump.cs:line 342
   at Server.Network.MessagePump.Slice() in C:\ServUO\Server\Network\MessagePump.cs:line 136
   at Server.Core.Main(String[] args) in C:\ServUO\Server\Main.cs:line 674`

**Comment by Iomega0318** [comment]
Also if you switch race then back to human changing the mannequin to male still shows the female gump but male body in world, if you redeed it and place again it will fix it.

**Comment by Iomega0318** [comment]
Fixed the crash, change this:
```
                protected override void OnTarget(Mobile from, object targeted)
                {
                    from.SendGump(new MannequinCompareGump(_Mannequin, (Item)targeted));
                }
```
to this:
```
                protected override void OnTarget(Mobile from, object targeted)
                {
                    if (targeted is Item item)
                    {
                        from.SendGump(new MannequinCompareGump(_Mannequin, item));
                    }
                    else
                    {
                        from.SendLocalizedMessage(1149667); // Invalid target.
                    }
                }
```

**Comment by pyro17** [comment]
Already fixed and merged by PR #4971
